bloc-did-method.md

```diff
@@ -8,99 +8,165 @@ The `did:bloc` DID method allows groups of independent organizations to share cu
+- Consortium: a group of Stakeholder organizations which all manage DIDs with their own Sidetree nodes, all on the same ledger. A Consortium hosts the configuration files which a resolver uses when first connecting to the network.
+- Stakeholder organization (or just stakeholder): an independent organization which follows some cooperative rules with the other stakeholder organizations within the same consortium to allow them all to manage DIDs using their own Sidetree nodes on the same ledger, and allow resolvers to manage DIDs through any of the stakeholders. Stakeholders host ledger nodes, host sidetree endpoints, and sign policy changes.
 
 #### `did:bloc` DID Format
 
 A DID for the Block DID Method has the following format:
 
-`did:bloc:[host]:[doc hash]`
+`did:bloc:[host]:[doc ID]`
+
+Where `[host]` is a URL host (eg, `foo.example.net`) and `[doc ID]` is the document ID as used in Sidetree.
 
-Where `[host]` is a URL host (eg, `foo.example.net`) and `[doc hash]` is the document hash as used in Sidetree.
+#### Data Model
 
-#### Consortium Data Model
+##### Consortium Files
 The policies of a consortium, along with information identifying the stakeholders, is stored under the consortium's domain, within `http://[consortium domain]/.well-known/did-bloc`
 
 `.well-known/did-bloc` should have this file hierarchy:
 
     .well-known/did-bloc/
-        consortium/
-            conf.json
-        stakeholders/
+        [domain].json
         history/
-    
-##### The `consortium` directory
-This directory contains the current consortium configuration, according to this particular server.
-
-###### `conf.json`
-The consortium config contains the following:
-- The domain name of the consortium
-- Consortium policy configuration settings
-- A list of stakeholders - containing, for each stakeholder:
-  - The web domain where its configuration can be found
-  - A local hashlink to the cached stakeholder configuration
-- A list of detached JWS signatures, from the stakeholders, on the config file
-- A local hashlink to the previous version of this config file
+
+`[domain].json`:
+This file contains the current consortium configuration, according to this particular server.
+
+`history`: contains the history database.
+
+##### Consortium Config Files
+A consortium config file is a concatenated JSON file:
+The first element is a JSON object containing:
+  - The domain name of the consortium
+  - Consortium policy configuration settings
+  - A list of stakeholders - containing, for each stakeholder:
+    - The web domain where its configuration can be found
+    - The did:bloc DID of the stakeholder, with the associated DID doc in Sidetree on the consortium ledger
+  - The hash of the previous version of this config file
```

Comment:
hash or hashlink?

Reply:
Technically not a hashlink, since it doesn't use a URL to link the resource - and I think it doesn't need to, since with a standard location to find the historical data, the hash can be all that's needed.

Especially with what Troy suggested for deployment, where the data is backed by a CAS.